### PR TITLE
DBDAART-3830_claims_files_BENE_COINSRNC_AMT_vs_TOT_BENE_COINSRNC_PD_AMT

### PR DESCRIPTION
**Pull request by FICU-tensileai**
## What is this and why are we doing it?
TMSIS documentation says these field names will be changed/is changed.  Change is applicable to all claim file types. These changes including IP, LT, OT, RX. 

Field Names include:
BENEFICIARY-COPAYMENT-AMOUNT: (HEADER LEVEL) BENE_COPMT_AMT -> TOT_BENE_COPMT_PD_AMT

BENEFICIARY-COINSURANCE-AMOUNT: (HEADER LEVEL) BENE_COINSRNC_AMT -> TOT_BENE_COINSRNC_PD_AMT

BENEFICIARY-DEDUCTIBLE-AMOUNT: (HEADER LEVEL) BENE_DDCTBL_AMT -> TOT_BENE_DDCTBL_PD_AMT

* Link to the Jira ticket for this change: https://jiraent.cms.gov/browse/DBDAART-3830

## What are the security implications from this change?
No security changes. We are simply changing field names to match what's coming in / will come in from TMSIS

## How did I test this?
https://cms-dataconnect-val.cloud.databricks.com/?o=955724715920583#job/937342215595277/run/439132410845135
https://cms-dataconnect-val.cloud.databricks.com/?o=955724715920583#job/635157112873553/run/1017579337942342
https://cms-dataconnect-val.cloud.databricks.com/?o=955724715920583#job/492738489039926/run/143190478775152
https://cms-dataconnect-val.cloud.databricks.com/?o=955724715920583#job/107254134537370/run/767784884280469

## Should there be new or updated documentation for this change? (Be specific.)
No, we've matched the TMSIS documentation

## PR Checklist
- [x] The JIRA ticket number and a short description is in the subject line
- [x] My code follows any applicable [style guides](https://cms-dataconnect.atlassian.net/wiki/search?text=style%20guide)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by C1KB-tensileai — APPROVED** [review]
Looks good @FICU-tensileai, I reviewed job runs for successful completing and exported standard output to search for incorrect column name references. None were found. Output here:
Search "BENE_DDCTBL_AMT" (0 hits in 0 files of 1 searched)
Search "BENE_COINSRNC_AMT" (0 hits in 0 files of 1 searched)
Search "BENE_COPMT_AMT" (0 hits in 0 files of 1 searched)

File: taf/IP/IP_Metadata.py [hunk 1]
```diff
@@ -579,7 +579,7 @@ def plan_id_num(colname: str, alias: str):
         "PTNT_CNTL_NUM",
         "HLTH_CARE_ACQRD_COND_CD",
         "PTNT_STUS_CD",
-        "cast(BIRTH_WT_GRMS_QTY as numeric(12,3)) as BIRTH_WT_GRMS_QTY",
+        "cast(BIRTH_WT_GRMS_QTY as decimal(12,3)) as BIRTH_WT_GRMS_QTY",
```
Comment (RIE9-adhoc) [inline taf/IP/IP_Metadata.py:582]: @FICU-tensileai  This data type change doesn't seem to be referenced in the ticket. Could you explain why this change was made?

Comment (FICU-tensileai) [reply taf/IP/IP_Metadata.py:582]: Good catch. Sorry. I was trying to debug one of the run failures and it was a suggestion from somewhere to see if that was causing it. It was not and I forgot to remove it. Removing it now.

Comment (FICU-tensileai) [reply taf/IP/IP_Metadata.py:582]: https://cms-dataconnect-val.cloud.databricks.com/?o=955724715920583#job/492738489039926/run/912910883953726
Removed and completed run to show it's still working without it.

**Review by RIE9-adhoc — APPROVED** [review]

**Review by C1KB-tensileai — APPROVED** [review]
